### DISCUSSION TITLE
How do I stop loaded files from being listed and treated as multiple files?

### DISCUSSION THREAD

**Discussion by atimholt** [enzo1982/freac#438] [Help & Support]
I have 27 flac files carefully and painstakingly assembled from ~50 hours of audio that were originally split into ~3 minute tracks on CDs (this is an audiobook backup/conversion project). Now that I've got my long-term, well organized flac "masters", I want to convert to a more practical format (opus, incidentally). However, the moment I load my 27 tracks into the joblist, the old track metadata is pulled up, unwanted—re-splitting the entire project into its (several hundred) original files. The whole point of combining the files was to end up with combined files. How do I treat combined files as combined files?

(I tried searching for this topic, but I might very easily have just not come up with the right key words for a proper search.)

**Comment by enzo1982** [comment]
The original CD track boundaries appear to be included in the FLAC metadata as chapter marks.

You can configure fre:ac to ignore chapters when loading files:

- In the configuration dialog, open the _Tags_ page.
- On the _Tags_ page, change to the _Other_ tab.
- Uncheck the _Read chapters from files_ option there.

These instructions are for fre:ac 1.1.6. In recent continuous builds, the option was moved to the _Metadata->Tag fields_ page of the configuration dialog.

**Reply by fairking** [reply]
> the option was moved to the `Metadata` -> `Tag fields` -> `Read chapters from files` of the configuration dialog.

Thanks @enzo1982 . Question. If I Uncheck `Read chapters from files` would then the converted file contain chapters? Or they will be gone?
I would like to convert one single file which is an album, but to keep chapters. Would it work like that? I mean I have many such files.
